biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -43,20 +46,45 @@ export const InputPage = () => {
   return (
     <div className="input-page">
       <form onSubmit={(e: React.SyntheticEvent<EventTarget>) => onSubmit(e)}>
-        <DatePicker
-          showPopperArrow={false}
-          selected={startDate}
-          onChange={(date: Date) => setStartDate(date)}
-        />
-        <DatePicker
-          showPopperArrow={false}
-          selected={endDate}
-          onChange={(date: Date) => setEndDate(date)}
-        />
-        {smallBatteryCellInput}
-        {largeBatteryCellInput}
-
-        <input type="submit" />
+      <h1>Welcome to the Biogrid Simulator!</h1>
+      <img src={Visual} alt="cannot display image"></img>
```

Comment:
alt should be used to explain what the image was supposed to show 
sometimes this alt text can be used by screenreaders to announce to blind/hard of seeing users what the image shows, and a description of "cannot display image" isn't very helpful :)

Reply:
sorry about that I actually had that there to test whether the image loaded or not because I was having trouble loading it in